build.sbt

```diff
@@ -330,6 +331,60 @@ lazy val tests: ProjectMatrix = (projectMatrix in file("tests"))
   )
   .dependsOn(core, circeJson, cats)
 
+
+val akkaHttpVanilla       = taskKey[Unit]("akka-http-vanilla")
+val akkaHttpTapir         = taskKey[Unit]("akka-http-tapir")
+val akkaHttpVanillaMulti  = taskKey[Unit]("akka-http-vanilla-multi")
+val akkaHttpTapirMulti    = taskKey[Unit]("akka-http-tapir-multi")
+val http4sVanilla         = taskKey[Unit]("http4s-vanilla")
+val http4sTapir           = taskKey[Unit]("http4s-tapir")
+val http4sVanillaMulti    = taskKey[Unit]("http4s-vanilla-multi")
+val http4sTapirMulti      = taskKey[Unit]("http4s-tapir-multi")
+def genPerfTestTask(servName: String, simName: String) = Def.taskDyn {
+  Def.task {
+    (Compile / runMain).toTask(s" perfTests.${servName}Server").value
+    (Gatling / testOnly).toTask(s" perfTests.${simName}Simulation").value
+  }
+}
+
+lazy val perfTests: ProjectMatrix = (projectMatrix in file("perf-tests"))
+  .enablePlugins(GatlingPlugin)
+  .settings(commonJvmSettings)
+  .settings(
+    name := "tapir-perf-tests",
+    libraryDependencies ++= Seq(
+      "io.gatling.highcharts" % "gatling-charts-highcharts" % "3.6.1" % "test",
+      "io.gatling"            % "gatling-test-framework"    % "3.6.1" % "test",
+      "com.typesafe.akka" %% "akka-actor-typed" % Versions.akkaStreams,
+      "com.typesafe.akka" %% "akka-stream-typed" % Versions.akkaStreams,
```

Comment:
do we need these -typed dependencies? IO think akka-http & akka-streams should be enough